app/helpers/categories_helper.rb

```diff
@@ -0,0 +1,21 @@
+module CategoriesHelper
+  def category_name
+    if @subcategory && @category
+      @subcategory.name
+    else
+      @category.name
+    end
+  end
+
+  def category_breadcrumb
+    output = ""
+    output += link_to "Home", root_path
+    output += "<span>&nbsp;»&nbsp;</span>"
+    output += link_to @category.name, category_path(@category.id)
```

Comment:
**Nitpick - Code Quality**
Ruby provides several ways to define multi-line strings. You can make use of the `\` operator provided or others mentioned in the resource below.

Check out this resource and make the necessary adjustments
https://cbabhusal.wordpress.com/2015/10/06/ruby-multiline-string-definition/